README.md

```diff
@@ -28,7 +28,6 @@ The combine_vcfs.py script takes individual sample vcfs and combines them into o
 
 Required inputs:
 * participant-data: Participant data (the downloaded data tab from Terra), should be a tab-delimited file with at minimum columns for 'entity:participant_id'(sample name with prohibited-characters replaced with underscores), 's'(sample name), and VCF output (path to the VCF output by mutect, name of this column is supplied to the vcf_col_name parameter)
-* participants-to-subset: Path to txt file of participant_ids to which the data should be subset (file should contain header and one line for each participant_id matching the 'entity:participant_id's supplied in Terra
```

Comment:
Why isn't this needed anymore?

Reply:
It's just no longer required. Everything in the participant-data file will be merged and providing a subset to use is optional.